d2renderers/d2svg/table.go

```diff
@@ -12,13 +13,42 @@ import (
 	"oss.terrastruct.com/util-go/go2"
 )
 
+func tableHeaderBorderRadius(shape d2target.Shape) string {
+	box := geo.NewBox(
+		geo.NewPoint(float64(shape.Pos.X), float64(shape.Pos.Y)),
+		float64(shape.Width),
+		float64(shape.Height),
+	)
+	topX, topY := box.TopLeft.X+box.Width, box.TopLeft.Y
+	out := fmt.Sprintf(`<clipPath id="%v">`, shape.ID)
```

Comment:
so, there's this kind of annoying thing:

when you have multiple SVGs in the same page, they can reference each other's IDs.

So mask IDs, for example we have to use the diagram hash (https://github.com/donglixiaoche/d2/blob/833967a296c5abb8b2b0d69ff898ca2fe520bbdf/d2renderers/d2svg/d2svg.go#L1650).

That way, when you embed two D2 diagrams in the same page, they don't conflict.

i think you have to do this for clip paths too. So on top of the shape.ID, you should append the diagram hash too.

This is actually a problem in our theme CSS right now (if you embed two diagrams rendered with different themes, the second one will overwrite the first ones): https://github.com/terrastruct/d2/issues/972

Reply:
fixed, clipPath id now is made up with `labelMaskID-shapeID` form

Reply:
so, masks used here is just invisible areas. for connection labels, we put a mask so that connections don't overlap with the text.

but imagine in this doc, where there's two SVGs, if we called both masks ID "123". In the first SVG, mask 123 is defined as one rectangle, and in the second, it's defined as another rectangle. Well then because they're on the same DOM, the second mask overrides the first mask, and both connections from unrelated diagrams would be using the same mask! 

<img width="952" alt="Screen Shot 2023-03-09 at 2 43 45 PM" src="https://user-images.githubusercontent.com/3120367/224177128-518ad214-fea2-40bf-b956-8b5531a27c8c.png">

Reply:
copy, thx